src/Symfony/Component/Notifier/Bridge/GatewayApi/.gitattributes

```diff
@@ -0,0 +1,4 @@
+/Tests export-ignore
+/phpunit.xml.dist export-ignore
+/.gitignore export-ignore
+/.gitattributes export-ignore
```

Comment:
tiny detail: Please use same order as all other component
```
/Tests export-ignore
/phpunit.xml.dist export-ignore
/.gitattributes export-ignore
/.gitignore export-ignore
```

Reply:
Thank you @jderusse.